app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: :100: 

- [ ] As stated in the project requirement, new users need to be able to confirm their emails before they can log in, you haven't enabled this feature currently, You need to enable `:confirmable` in your `user.rb` file. I'll suggest your team work on this. :+1: :+1: 

- - Screenshot reference:
![Selection_014](https://github.com/rbd3/Blog-App/assets/107632780/5da1e75a-9112-4aad-bf31-88c2187569ac)


- [ ] Also as stated in the project requirement, you need to work on the devise login view to look like the wireframe. :+1: I'll suggest you add the appropriate styling to achieve this. :smile: 

- - Wireframe:
![Screenshot from 2023-10-04 15-28-09](https://github.com/rbd3/Blog-App/assets/107632780/bbb77aa1-b457-4dce-9435-ee4c068b3257)